### DISCUSSION TITLE
Try it on StackBlitz

### DISCUSSION THREAD

**Discussion by SompalSingh** [twbs/bootstrap#39206] [Q&A]
After building Bootstrap 5.2.2, it has provided the `_site` repository. However, in this repository, the "Try it on StackBlitz" functionality is not working.

**Comment by julien-deramond** [comment]
What's the context? Are you trying to build Bootstrap locally to contribute?

If so, you only need to run the following after having checked out our _main_ branch:

```
npm i
npm run start
```

And go to http://localhost:9001/docs/5.3/components/accordion/ (for instance), and the "Edit on StackBlitz".
